README.md

```diff
@@ -70,6 +70,19 @@ cmake -G "Visual Studio 17 2022 Win64" -B build
 # Generates MSVS solution build/LuaBridge.sln
 ```
 
+## Installing LuaBridge(vcpkg)
```

Comment:
A space before the opening bracket?

Reply:
The standardized format can help users to check faster.  I will modify it. :)